plex/api.py

```diff
@@ -663,13 +676,28 @@ def fillVideoInfos(
 
         if mediaPart:
             # extract the absolute / actual path and the stream URL from the selected MediaPart
-            info['path'] = mediaPart.file
-            item.setPath(plexServer.url(mediaPart.key, includeToken=True))
+            info['path'] = mediaPart.file  # path to file on disk
+
+            # determine if directPlay is enabled and possible
+            if allowDirectPlay:
+                log("Configuring item for direct play", xbmc.LOGDEBUG)
```

Comment:
This log is not very helpful since it doesn't even mention the specific item. But I would drop it completely because it will generate a huge log spam when importing large libraries.